setup/gcp/installer/util/utils.py

```diff
@@ -298,3 +298,168 @@ def sanitize_conf_values(conf_values):
         if not conf_values[key]:
             conf_values[key] = '""'
     return conf_values
+
+
+def merge_object(base_obj, target_obj,
+                 fields_to_ignore=None, field_identifiers=None):
+    """Merge objects.
```

Comment:
Can you please add some high-level description for what the merging strategy is?  For example, as we discussed, if user has 8 additional custom rules and do not match the base rules, how do they end up being handled.?  This would also help me to understand if the actual implementation matches the the merging strategy.